### PR TITLE
Minor edit to T81 and S123

### PR DESCRIPTION
**Pull request by Jianing-Song**
For S123: I changed "$i$th prime" to "$(i+1)$th prime" since natural number starts with $0$, and $2$ is generally seen as the first prime. Please correct me if this edit is unnecesary/unreasonable.

For T81, I added a proof for the assertion. But still I have the following questions:
1. In {{mathse:3143628}}, I gave the question ID here. Should I give the answer ID (3144337) instead?
2. The proof uses the fact that regular + Lindelof => normal. Should this result be mentioned in T81, so we say something like "See {{mathse:3143628}} for a proof, which uses {T30}"?
3. I very much hope to mention S90 in T81, because it shows that we cannot have better results than uncountability. Could we add a sentence like "Note that we cannot do better, as {S90} is a {P36} (connected) {P5} ($T_3$) space satisfying {P114} (cardinality $=\aleph_1$), the smallest uncountable cardinal number"?

### CODE REVIEW

File: spaces/S000123/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
 ---
-Let $\{C_i\ |\ i \in \omega\}$ be a collection of disjoint dense subsets of $\mathbb{Q}$. For specificity, let \(p(i)\) be the \(i\)th prime,
+Let $\{C_i\ |\ i \in \omega\}$ be a collection of disjoint dense subsets of $\mathbb{Q}$. For specificity, let \(p(i)\) be the \((i+1)\)th prime,
 and let \(C_i\) be the rational numbers of the form $\frac{a}{p(i)^n}$, with $a$ an integer not divisible by $p(i)$.
```
Comment (StevenClontz) [inline spaces/S000123/README.md:10]:
```suggestion
Let $\{C_i\ |\ i < \omega\}$ be a collection of disjoint dense subsets of $\mathbb{Q}$. For specificity, let \(\{p_i:i<\omega\}\) enumerate the prime numbers,
and let \(C_i\) be the rational numbers of the form $\frac{a}{p_i^n}$, with $a$ an integer not divisible by $p_i$.
```

Comment (StevenClontz) [reply spaces/S000123/README.md:10]: Another possible tweak to consider, avoiding the ambiguity of what the 0th prime number might be.

**Comment by StevenClontz** [comment]
> In {{mathse:3143628}}, I gave the question ID here. Should I give the answer ID (3144337) instead?

Using the answer ID is better I think.

> The proof uses the fact that regular + Lindelof => normal. Should this result be mentioned in T81, so we say something like "See {{mathse:3143628}} for a proof, which uses {T30}"?

I wouldn't - we sometimes tweak theorems to strengthen them (maybe we shouldn't). But this would cause `{T30}` to possibly change later. So I'd leave this be for now.

> I very much hope to mention S90 in T81...

Sure, I suggested one way to do this above.

File: theorems/T000081.md [hunk 2]
```diff
@@ -10,7 +10,10 @@ then:
 refs:
 - doi: 10.1007/978-1-4612-6290-9
   name: Counterexamples in Topology
+- mathse: 3143628
+  name: Showing every connected regular space having more than one point is uncountable without using proof by contradiction
 ---
 
+See {{mathse:3143628}} for a proof.
```
[the remark this answers is not in the document]
Comment (prabau) [reply theorems/T000081.md:17]: Good to have this note, but can we hint at the connection with the result a little better?  As originally suggested, how about something like 
"(Note that one cannot conclude a stronger cardinality result, as {S90} is an example of a {P36} {P5} space satisfying {P114}.)"

**Comment by prabau** [comment]
T81: in the `refs:` section, can we put the mathse reference before the doi reference, as the doi one is less important here?

**Comment by prabau** [comment]
Actually, looking at Henno Brandsma's proof of T81 using the Lindelof property, I am wondering if T81 is even needed.  It seems it could possibly be deduced from other results already, since pi-base shows that the combination

  connected + T3 + multiple points + $|X|<\mathfrak c$ + Lindelof

is impossible: [π-Base, Search for `connected+T3+multiple points+Cardinality $\lt\mathfrak c$+Lindelof`](https://topology.pi-base.org/spaces?q=connected%2BT3%2Bmultiple+points%2BCardinality+%24%5Clt%5Cmathfrak+c%24%2BLindelo)
and the justification does not seem to use T81.

What do you think?  Should we try to remove it and see what happens?  And if so, any reason it should be kept anyway?

**Comment by Jianing-Song** [comment]
Indeed, Countable ∧ Connected ∧ $T_3$ ∧ Has multiple points is impossible by the remaining theorems after I deleted T81.

For record, the content of T81 was as follows:

> ---
> uid: T000081
> if:
>   and:
>   - P000036: true
>   - P000005: true
>   - P000125: true
> then:
>   P000057: false
> refs:
> - mathse: 3144337
>   name: Showing every connected regular space having more than one point is uncountable without using proof by contradiction
> - doi: 10.1007/978-1-4612-6290-9
>   name: Counterexamples in Topology
> ---
> 
> See {{mathse:3144337}} for a proof.
> 
> Asserted on page 223 of {{doi:10.1007/978-1-4612-6290-9}}.
> 
> (Note that one cannot conclude a stronger cardinality result, as {S90} is an example of a {P36} {P5} space satisfying {P114}.)
>

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**